stable/victoria-metrics/templates/vminsert-deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
     {{- end }}
       containers:
         - name: vminsert
-          image: "{{ .Values.vminsert.image.repository }}:{{ .Values.vminsert.image.tag }}"
+          image: "{{ .Values.vminsert.image.repository }}:v{{ .Chart.AppVersion }}"
```

Comment:
Whats the point to do templating with additional symbol here? Can we use define appVersion as "v1.35.4-cluster".

Reply:
Well... this is more for having consistent output in the `APP VERSION` column of the `helm list` command:

```
NAME                 	REVISION	UPDATED                 	STATUS  	CHART                       	APP VERSION   	NAMESPACE  
af-grafana-prod      	14      	Wed Apr 15 17:29:12 2020	DEPLOYED	grafana-3.11.0              	6.4.2         	grafana    
cerebro              	1       	Mon Sep  9 11:34:40 2019	DEPLOYED	cerebro-1.2.0               	0.8.4         	default    
cert-issuer          	2       	Tue Oct 29 16:12:03 2019	DEPLOYED	cert-issuer-3.0.1           	v0.11.0       	ingress    
cert-manager         	1       	Mon Oct 28 17:25:14 2019	DEPLOYED	cert-manager-v0.11.0        	v0.11.0       	ingress    
efk                  	94      	Fri May 15 12:06:10 2020	DEPLOYED	efk-1.1.0                   	              	default    
elastic              	90      	Thu Apr  2 14:53:54 2020	DEPLOYED	elasticsearch-1.1.0         	6.7.1         	default    
elastic-hq           	1       	Thu Jun  6 15:55:52 2019	DEPLOYED	elastichq-0.1.0             	3.5.0         	default    
es-curator-close-idx 	11      	Fri Jan 24 14:13:12 2020	DEPLOYED	elasticsearch-curator-2.0.3 	5.7.6         	default    
es-curator-create-idx	7       	Fri Jan 24 14:13:12 2020	DEPLOYED	elasticsearch-curator-2.0.3 	5.7.6         	default    
es-curator-delete-idx	8       	Fri Jan 24 14:13:10 2020	DEPLOYED	elasticsearch-curator-2.0.3 	5.7.6         	default    
es-curator-shrink    	32      	Fri Jan 24 14:13:11 2020	DEPLOYED	elasticsearch-curator-2.0.3 	5.7.6         	default    
es-curator-snap      	17      	Fri Jan 24 14:13:10 2020	DEPLOYED	elasticsearch-curator-2.0.3 	5.7.6         	default    
es-exporter          	4       	Tue Jan 28 14:49:20 2020	DEPLOYED	elasticsearch-exporter-2.2.0	1.1.0         	default    
kibana-antiblocking  	2       	Thu Apr 16 19:07:57 2020	DEPLOYED	kibana-2.3.1                	6.7.0         	default    
kibana-ar            	2       	Thu Apr 16 19:07:56 2020	DEPLOYED	kibana-2.3.1                	6.7.0         	default    
kibana-data          	2       	Thu Apr 16 19:07:58 2020	DEPLOYED	kibana-2.3.1                	6.7.0         	default    
kibana-devops        	2       	Thu Apr 16 19:07:57 2020	DEPLOYED	kibana-2.3.1                	6.7.0         	default    
kibana-elite         	2       	Thu Apr 16 19:07:56 2020	DEPLOYED	kibana-2.3.1                	6.7.0         	default    
kibana-partners      	2       	Thu Apr 16 19:07:55 2020	DEPLOYED	kibana-2.3.1                	6.7.0         	default    
kibana-qa            	2       	Thu Apr 16 19:07:53 2020	DEPLOYED	kibana-2.3.1                	6.7.0         	default    
kibana-secops        	2       	Thu Apr 16 19:07:58 2020	DEPLOYED	kibana-2.3.1                	6.7.0         	default    
kibana-server        	2       	Thu Apr 16 19:07:57 2020	DEPLOYED	kibana-2.3.1                	6.7.0         	default    
nginx-ingress        	1       	Mon Oct 28 17:20:21 2019	DEPLOYED	nginx-ingress-1.6.16        	0.24.1        	ingress    
okta-proxy-aab       	4       	Tue Aug 20 14:14:30 2019	DEPLOYED	https-okta-proxy-0.3.1      	0.5.14.1      	default    
okta-proxy-ar        	4       	Tue Aug 20 14:14:31 2019	DEPLOYED	https-okta-proxy-0.3.1      	0.5.14.1      	default    
okta-proxy-data      	4       	Tue Aug 20 14:14:31 2019	DEPLOYED	https-okta-proxy-0.3.1      	0.5.14.1      	default    
okta-proxy-devops    	4       	Tue Aug 20 14:14:31 2019	DEPLOYED	https-okta-proxy-0.3.1      	0.5.14.1      	default    
okta-proxy-elite     	4       	Tue Aug 20 14:14:31 2019	DEPLOYED	https-okta-proxy-0.3.1      	0.5.14.1      	default    
okta-proxy-partners  	4       	Tue Aug 20 14:14:30 2019	DEPLOYED	https-okta-proxy-0.3.1      	0.5.14.1      	default    
okta-proxy-qa        	4       	Tue Aug 20 14:14:32 2019	DEPLOYED	https-okta-proxy-0.3.1      	0.5.14.1      	default    
okta-proxy-secops    	4       	Tue Aug 20 14:14:31 2019	DEPLOYED	https-okta-proxy-0.3.1      	0.5.14.1      	default    
okta-proxy-server    	4       	Tue Aug 20 14:14:32 2019	DEPLOYED	https-okta-proxy-0.3.1      	0.5.14.1      	default    
prom-ha-proxy        	3       	Thu Oct  3 15:14:00 2019	DEPLOYED	prometheus-ha-proxy-0.2.0   	0.1.0-alpha.1 	grafana    
prometheus-efk       	22      	Fri Aug  2 17:09:46 2019	DEPLOYED	prometheus-efk-1.0.0        	2.11.1        	monitoring 
rds-ca-cert          	1       	Wed Jan 15 16:28:02 2020	DEPLOYED	rds-ca-cert-1               	2019          	grafana    
threatstack          	2       	Tue Apr 28 17:17:33 2020	DEPLOYED	threatstack-agent-1.0.1     	2.1.3         	threatstack
victoria-metrics     	7       	Thu Apr 30 14:27:04 2020	DEPLOYED	victoria-metrics-0.15.0     	1.35.0-cluster	tsdb       
```

As you see, there is no any superfluous symbols there.

Reply:
cert-manager and cert-issuer didn't count?